### ISSUE TITLE
Divine Soul Sorcerer Spells

### ISSUE THREAD

**Issue by eze783** [larrychristensen/orcpub#50]
When building a Divine Soul Sorcerer, the Cleric class spells do not show as available.

**Comment by Heliosmaster** [comment]
@eze783 Divine Soul sorcerer is not part of the SRD so it cannot be in the code of orcpub. If you are referring to some content that you made using the "Homebrew" tools you should report this issue to the creator of the homebrew file that you are using. :(

**Comment by codeGlaze** [comment]
Homebrewing the class under current "limitations" of OP2 leaves a relatively unfinished product. It's over if the most requested items but a complete nfhtmare to implement afaict because of how it selectively mixes arcane and divine casting.

There's not much a homebrewer can do to make it feature-complete.

**Comment by Heliosmaster** [comment]
Yes, it's pretty hard to make a system can generalize all rules like "Oh by the way you can have the spell of class X and this other feature from class Y".  :(
